views/templates/steps/backup-options.html.twig

```diff
@@ -41,6 +41,7 @@
           buttonLabel: 'Download backup logs'|trans({}),
           buttonUrl: download_path,
           buttonDownload: filename,
+          buttonTrack: 'Backup logs downloaded'
```

Comment:
These events could reuse the constants defined in a PHP class, to avoid duplicating the same event name in different places (Currently duplicated in LogsService.php)